### ISSUE TITLE
Color contrast does not meet AA level standards in the danger button when hovering over it

### ISSUE THREAD

**Issue by ltchou** [BPbase/piman#9]
**Describe the bug**
Color contrast < 4.5 when hovering over the danger button

**To Reproduce**
1. Go to https://bpbase.github.io/piman/demo/#/button
2. Mouse over 'Danger' theme button
3. The color contrast between text color (AF0404) and background (FEB4B4) is 4.36 < 4.5

**Expected behavior**
Should be > 4.5 

**Screenshots**
![image](https://github.com/BPbase/piman/assets/6311615/08e93452-0654-4238-9857-7ad0f0929442)

**Desktop (please complete the following information):**
 - Browser:  chrome
 - Version: 124.0.6367.208
